### DISCUSSION TITLE
Monailabel training the data separately

### DISCUSSION THREAD

**Discussion by srivatsan2607** [Project-MONAI/MONAILabel#1600] [Q&A]
HI, 
I have annotated data of images of type .nii.gz files in a folder. I want to train all the images under the folder for segmentation model and use this trained model to run auto segmentation on new images. It would be really helpful if you could provide solution or guidance on how to train the model separately with annotated data. 

Is it possible to train the model with images from the api route: /train/{model} Run Training Task for specific model
Thank you

**Comment by diazandr3s** [comment]
Hi @srivatsan2607,

To train a model from scratch, please follow [these](https://www.youtube.com/watch?v=3HTh2dqZqew) or [these](https://youtu.be/-HAryYAO5J4?si=U7vozWTZyFun5u2-&t=565) instructions.

You can also find more information here: https://github.com/Project-MONAI/MONAILabel/discussions/1593#discussioncomment-7675003


Let us know how that goes,
